setup.py

```diff
@@ -71,6 +71,12 @@
             '2k_earth_daymap.jpg',
         ],
     },
+    download_url="https://pypi.python.org/pypi/pyvista",
+    project_urls={
+        "Bug Tracker": "https://github.com/pyvista/pyvista/issues",
+        "Documentation": "https://docs.pyvista.org/",
+        "Source Code": "https://github.com/pyvista/pyvista",
```

Comment:
This is very much just bikeshedding, but I would put documentation first.

Reply:
```suggestion
        "Documentation": "https://docs.pyvista.org/",
        "Bug Tracker": "https://github.com/pyvista/pyvista/issues",
        "Source Code": "https://github.com/pyvista/pyvista",
```